src/js/components/form-elements/LabelField.jsx

```diff
@@ -40,10 +42,9 @@ const LabelField = (props) => {
     return (
       <div className="form-group my-0">
         <Tooltip
-          html={(<div>{formattedValue}</div>)}
```

Comment:
I think the formattedValue will need to be in div, if you change this to:
`html={tooltipValue || (<div>{formattedValue}</div>)}`
it should be fine